.github/workflows/build.yml

```diff
@@ -3,26 +3,15 @@ name: Install and build
 on:
   pull_request:
     branches: [ master ]
-  workflow_call:
+  workflow_dispatch:
```

Comment:
This should not be changed, otherwise `deploy.yml` will break.

But honestly, we should probably remove the dependency on build from deploy anyway, since the deploy flow basically does anything the build does anyway, and there is no harm done by doing it on the server of it would fail

Reply:
Alright, seems reasonable! Removed the dependency now!